reference.md

```diff
@@ -1253,26 +1274,30 @@ from regular tables defined with square or curly brackets. Similarly symbols
 are tables with a string entry for their name and a marker metatable. You
 can use `tostring` to get the name of a symbol.
 
-* `list` - return a list, which is a special kind of table used for code
-* `sym` - turn a string into a symbol
-* `list?` - is the argument a list?
-* `sym?` - is the argument a symbol?
-* `table?` - is the argument a non-list table?
+* `list` - return a list, which is a special kind of table used for code.
+* `sym` - turn a string into a symbol.
+* `gensym` - generates a unique symbol for use in macros, accepts an optional prefix string.
+* `<symbol>#` - generates a unique symbol, derived from the given symbol.
```

Comment:
@andreyorst `gensym` & `<symbol>#`?

Reply:
This list is intended to show all the functions which are available in compile scope. While auto-gensym notation is useful and needs to be explained, it should be explained somewhere else.